src/ui/connection_profiles.ui

```diff
@@ -2220,6 +2220,12 @@
                 <pointsize>9</pointsize>
                </font>
               </property>
+              <property name="accessibleName">
+               <string>Profile name</string>
+              </property>
+              <property name="accessibleDescription">
+               <string comment="Using lower case letters for 'ASCII' may make speech synthesisers say 'askey' which is quicker than 'Aay Ess Cee Eye Eye'!">A unique name for the profile but which is limited to a subset of ascii characters only.</string>
+              </property>
```

Comment:
Even though it's nice you think of how Mudlet will affect speach software, I'd drop the whole mentioning of ascii subset. This is too much detail here, and it is also hidden.

Reply:
A *whole lot* is going to be hidden if you are VI or blind - lets not make it *too* hard for them to name their ~~MUD~~ game profiles. Depending on how someone configures their screen reader the *accessibleDescription* is the detailed explanation that is given after the other salient details are announced - so IMHO this is not so unreasonable as it might seem. Easier to say what is needed *before* hand rather then having to explain in an error message afterwards...